
builtin_interfaces/Time stamp

float32 pitch_angle
float32 angular_velocity
float32 pitch_angle_setpoint
float32 tilt_angle
